DataFormats/GeometryVector/interface/Pi.h

```diff
@@ -28,13 +28,13 @@
 
 namespace Geom {
 
-  inline double pi()     {return     3.141592653589793238;}
-  inline double twoPi()  {return 2. *3.141592653589793238;}
-  inline double halfPi() {return 0.5*3.141592653589793238;}
+  inline constexpr double pi()     {return     3.141592653589793238;}
+  inline constexpr double twoPi()  {return 2. *3.141592653589793238;}
+  inline constexpr double halfPi() {return 0.5*3.141592653589793238;}
```

Comment:
because is like this since orca times...
(somewhere else you will find CLHEP::pi())
I would not mind to go back to old good C macros
(as already done elsewhere)